src/trajectory_data_handling/sqlite_querying.cpp

```diff
@@ -8,9 +8,10 @@ namespace trajectory_data_handling {
     unsigned long query_handler::m_currentTrajectory = 1;
     int query_handler::m_order{};
     std::shared_ptr<std::vector<data_structures::Trajectory>> trajectory_data_handling::query_handler::all_trajectories;
```

Comment:
Er det muligt at omdøbe `allTrajectories` til `original_trajectories`? Nu hvor vi har splittet det op og har to containere, både den originale og den simplified, kan den så ikke få et mere præcist navn nu? 

```suggestion
    std::shared_ptr<std::vector<data_structures::Trajectory>> trajectory_data_handling::query_handler::original_trajectories;
```

Reply:
Lyder som en god ide, det er gjort